docs/user/reporting/reporting-troubleshooting.asciidoc

```diff
@@ -55,6 +55,8 @@ if it encounters errors, logs them in the Console. Unfortunately, the error mess
 information about why Chromium failed to run. The most common error message is `Error: connect ECONNREFUSED`, which indicates
 that {kib} could not connect to the Chromium process.
 
+Chromium is not compatible with ARM RHEL/CentOS.
```

Comment:
Suggest moving this below the paragraph "To troubleshoot the problem" .  Otherwise it interrupts the flow of the two paragraphs.

Does this information warrant being called out in a note?